src/test/java/com/github/tomakehurst/wiremock/recording/SnapshotStubMappingPostProcessorTest.java

```diff
@@ -57,21 +63,49 @@ public StubMapping apply(StubMapping stubMapping) {
 
     final List<StubMapping> actual =
         new SnapshotStubMappingPostProcessor(false, transformerRunner, null, null)
-            .process(TEST_STUB_MAPPINGS);
+            .process(testStubMappings);
 
     assertThat(actual, hasSize(2));
     assertThat(actual.get(0).getRequest().getUrl(), equalTo("/foo/transformed"));
     assertThat(actual.get(1).getRequest().getUrl(), equalTo("/bar/transformed"));
   }
 
   @Test
-  public void processExtractsBodiesWhenMatched() {
+  public void
+      process_withShouldRecordRepeatsAsScenariosAndTransformer_runsTransformerBeforeScenarioProcessor() {
```

Comment:
I'm finding these test names hard to make sense of, and they're inconsistent in style with the rest of the codebase. Please could we make them statements of outcome and camel cased?

Reply:
Apologies for going in a different direction with test naming here.

A bit on rationale, the pattern `method_condition_expectedOutcome` was chosen based on capturing the recommended components for unit test naming per [Google Testing](https://testing.googleblog.com/2014/10/testing-on-toilet-writing-descriptive.html) and specifically I've included outcomes in this context because documenting that aspect feels valuable.  The behavior of the `SnapshotStubMappingPostProcessor` is a bit subtle, I want to make sure each test's goal is articulated so the reader isn't left attempting to parse the intended behavior from the assertion set.

All of this can be done in a fashion more aligned to the conventions of this project.  I've switched over to a `method<With>condition<Should>expectedOutcome` format, so I think we're in good shape.